docs/reference/commandline/run.md

```diff
@@ -540,3 +541,9 @@ the three processes quota set for the `daemon` user.
 The `--stop-signal` flag sets the system call signal that will be sent to the container to exit.
 This signal can be a valid unsigned number that matches a position in the kernel's syscall table, for instance 9,
 or a signal name in the format SIGNAME, for instance SIGKILL.
+
+### Configure namespaced kernel parameters (sysctls) at runtime
+
+The `--sysctl` flag sets the namespaces kernel parameters (sysctls) in the container, specified as
+          `{ <name>: <Value> }`, for example:
+	  `{ "net.ipv4.ip_forward": "1" }`
```

Comment:
Re-reading this, can you make this a "full" example, as I'm currently in doubt if the `{}` should be used on the command line, or not

i.e. currently it reads that it should be used as:

    docker run --sysctl '{ "net.ipv4.ip_forward: "1" }' someimage

Is that correct? Or should it be:

    docker run --sysctl net.ipv4.ip_forward=1 someimage